english/water-dreams/pig-farm/love-bacon/love-bacon.md

```diff
@@ -1,3 +1,10 @@
 You take a moment to reflect and realize your psychological state is 
-more fragile than you thought possible for a pig. You decide to search
-for a fellow pig that you can commiserate with...
\ No newline at end of file
+more fragile than you thought possible for a pig.
+
+What do you do next?
+
+[Search for a fellow pig that you can commiserate with](pig-friend/pig-friend.md)
+
+[Search for a pig therapist](pig-therapist/pig-therapist.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.